.github/workflows/tests-cleanup.yml

```
@@ -0,0 +1,77 @@
+name: TestsCleanUp
+on:
+  push:
+    branches:      
+      - tf-*      
+  schedule:
+    - cron: '0 0 * * *' # Runs dayly at 12:00:00
+  workflow_dispatch:
+jobs:
+  build:    
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1    
```

We use the most recent version, and are explicit about versions for actions, and then Dependabot will keep them up-to-date thanks to https://github.com/Particular/NServiceBus.AmazonSQS/blob/master/.github/dependabot.yml#L16-L20 which is synced from RepoStandards.
```suggestion
    - uses: actions/checkout@v2.3.4
```